### ISSUE TITLE
Inclusion of Kali Linux 1.1.0

### ISSUE THREAD

**Issue by marciosilvarj** [SolidRun/ignition-imx6#10]
Could Kali Linux 1.1.0 be included?
Kind Regards,
Marcio

**Comment by rabeeh** [comment]
I have Kali scripts ready. Did you test it? What feedback can you say about its quality level?

**Comment by rabeeh** [comment]
Actually i'm looking at the download site and can't see 1.1.0 -
https://www.offensive-security.com/kali-linux-vmware-arm-image-download/

Is it published somewhere else?

**Comment by marciosilvarj** [comment]
I could make this version run on a Raspberry PI b+ http://images.offensive-security.com/1.0.9/kali-1.0.9-rpi.img.xz.
I tried this same image on HummingBoard-i2eX and it does not work.
I downloaded  http://cdimage.kali.org/kali-1.1.0/kali-linux-1.1.0-armel.img.xz and http://cdimage.kali.org/kali-1.1.0/kali-linux-1.1.0-armhf.img.xz. None of them worked on RPi or on HummingBoard-i2eX.

Kind regards,

Marcio
